README.md

```diff
@@ -200,3 +200,18 @@ If you see `ftype=1` then you have d_type support.
 The default cpu/memory/disk resources when using virtual machines are provided
 by the [vm_setup_vars.yml](vm_setup_vars.yml) file, which sets some dev-scripts
 variables that override the defaults in metal3-dev-env
+
+### Testing a custom machine-api-operator image with this deployment
+The script run-custom-mao.sh allows the machine-api-operator pod to be re-deployed
+with a custom image.
+- Turn on this functionality by setting TEST_CUSTOM_MAO to true.
+
+- Call ./run_custom_mao.sh from within 11_register_hosts.sh providing it with
+its parameters (use place holder already provided in 11_register_hosts.sh).
+
+For example:
+./run-custom-mao.sh <path in quay.io for the custom MAO image with tag> <repo name> <branch name>
+
+- All input parameters can be set via CUSTOM_MAO_IMAGE, REPO_NAME and MAO_BRANCH respectively.
```

Comment:
To make this clearer should we say "Alternatively, all input parameters can be set via variables ..." ?